content/test/integration/service/deployments/deployment-filters.spec.ts

```diff
@@ -168,13 +168,15 @@ describe('Integration - Deployment Filters', () => {
     const result: Timestamp[] = []
     for (const { deployData } of entities) {
       const newAuditInfo = { version: EntityVersion.V2, authChain: deployData.authChain, ...overrideAuditInfo }
-      const deploymentTimestamp = await service.deployEntity(
+      const deploymentTimestamp: Timestamp | ErrorList = await service.deployEntity(
         Array.from(deployData.files.values()),
         deployData.entityId,
         newAuditInfo,
         ''
       )
-      result.push(deploymentTimestamp)
+      if (typeof deploymentTimestamp === 'number') {
```

Comment:
If we use type guards correctly, then the `as Timestamp` shouldn't be necessary

Reply:
I couldn't make that work, if you could make the change proposal, that'd be great! 😄

Reply:
Let's try it after the `ErrorList` turns into an object